website_multi/models/website.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    # Replaces the web.base.url system parameter
```

Comment:
can you explain?

Reply:
For example to use in generating the sitemap.xml

Reply:
but why replace? Shouldn't you reuse it instead or use it as a default?
Is it just for the switcher?

Reply:
For now the switcher only uses this field. But I have another module website_multi_blog (no PR yet) that uses this field to generate the RSS for multiple website properly. I can still use the web.base.url property as default in that module.